### PR TITLE
Fix Keystone when using Crowbar Batch

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -4438,8 +4447,10 @@ puts y.to_yaml" > /root/keystone-test-endpoint-update.yaml
     safely crowbar batch --timeout 1500 build < /root/keystone-test-endpoint-update.yaml
 }
 
-function update_keystone_endpoint
+function test_keystone_toggle_ssl
 {
+    prepare_proposal_for_modification $proposal $proposaltype
```
Comment (sjamgade) [inline scripts/qa_crowbarsetup.sh]: I am unable to understand why this change.
`set_keystone_endpoint` creates a batch export in `/root/keystone-test-endpoint-update.yaml` and uses that for another batch run so the `$pfile` created by this call is not really used

and also I could not find `proposal` and `proposal_type` being set for the `testsetup` step

Comment (dirkmueller) [reply scripts/qa_crowbarsetup.sh]: If you look at the mkcloud logs you can see errors that it fails to read thr attributes of the ssl cert names because it uses accessor functions that read the $pfile directly. I believe the rest of the failures are a fallout of that.

Comment (dirkmueller) [reply scripts/qa_crowbarsetup.sh]: My tests are still running, will check tonight

**Comment by dirkmueller** [comment]
(needs final test prior merging)

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (sjamgade) [reply scripts/qa_crowbarsetup.sh]: okay, that makes sense. But in batch step these variables will not defined already, so you will end up having 
 ```11:14:36 ++(qa_crowbarsetup.sh:1977) get_proposal_filename(): echo /root/.default.proposal```

The gating job passed because it wasnt a batch job and so the file `keystone.default.proposal` was already there

Here you might want to make it 
``` prepare_proposal_for_modification keystone default ``` 

since we know it is `keystone` and `default`

Comment (dirkmueller) [reply scripts/qa_crowbarsetup.sh]: Correct, that was a copy&paste error
